test/machineslib.py

```diff
@@ -40,6 +40,8 @@ def performAction(self, vmName, action, checkExpectedState=True, connectionName=
             b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Running")
         if action == "forceOff":
```

Comment:
why not just doing `if action == "forceOff" or action == "off"` the same way as on line 39?

Reply:
Sure